integrations/transportation/trainline.js

```diff
@@ -0,0 +1,174 @@
+import { get } from 'lodash';
+import { CookieJar, CookieAccessInfo } from 'cookiejar';
```

Comment:
Hi @liamgarrison ,
Thanks for the help! Unfortunately I don't think cookiejar will work on ReactNative when we deploy to the app.
The reason is due to internals of how RN works, and I won't have time to test with cookiejar unfortunately.

Instead, you should use superagent as we did here:
https://github.com/tmrowco/tmrowapp-contrib/blob/master/integrations/transportation/rejsekort.js#L20
It will automatically handle cookies as if you were in a browser, so no need to get/set them.

Furthermore, this also means we can't re-use cookies across multiple app sessions, which means it won't work to store the cookiejar in the state. What we do instead (in rejsekort.js for example) is that we re-authenticate at each `collect()`.

Let me know if you have any questions

Olivier

Reply:
Ah ok. I see. Will change to use superagent